ext/standard/basic_functions.stub.php

```diff
@@ -59,3 +59,9 @@ function stream_wrapper_restore(string $protocol): bool {}
 
 /** @return int|false */
 function array_push(array &$stack, ...$args) {}
+
+/* {{{ base64.c */
+
+function base64_encode(string $str): string {}
+
+function base64_decode(string $str, bool $strict): string {}
```

Comment:
This returns false in the case of an error
```suggestion
/** @return string|false */
function base64_decode(string $str, bool $strict) {}
```

Reply:
Additionally `$strict` is an optional argument, so this should be `bool $strict = false`.